### PR TITLE
On 1397 financial copy changes

### PR DESCRIPTION
**Pull request by davidgouge**

### CODE REVIEW

File: src/SFA.DAS.ApplyService.Web/Views/Application/Pages/Index.cshtml [hunk 1]
```diff
@@ -50,11 +50,19 @@
...
+                @if (!string.IsNullOrEmpty(@Model.Title))
+                {
+                    <h1 class="govuk-heading-xl">
+                        @Html.Raw(Model.Title)
+                    </h1>
+                }
+
+                @if (!string.IsNullOrEmpty(@Model.BodyText))
+                {
+                    @Html.Raw(Model.BodyText)
```
Comment (MarkFCain) [inline src/SFA.DAS.ApplyService.Web/Views/Application/Pages/Index.cshtml]: should be in a div or something

Comment (davidgouge) [reply src/SFA.DAS.ApplyService.Web/Views/Application/Pages/Index.cshtml]: @gdbarnes knows what he's doing. ;)

Comment (davidgouge) [reply src/SFA.DAS.ApplyService.Web/Views/Application/Pages/Index.cshtml]: Or, more helpfully, the body text itself contains surrounding tags.  Hence the Html.Raw. 👍

Comment (gdbarnes) [reply src/SFA.DAS.ApplyService.Web/Views/Application/Pages/Index.cshtml]: Yeah.. I know what I'm doing. 100%. No question about it. Don't even ask! 😐

**Pull request merged**